src/net/STUN/STUNClient.cs

```diff
@@ -82,8 +82,19 @@ public static IPEndPoint GetPublicIPEndPoint(string stunServer, int port = DEFAU
                                         {
                                             STUNAddressAttribute stunAddress = (STUNAddressAttribute)stunAttribute;
                                             publicEndPoint = new IPEndPoint(stunAddress.Address, stunAddress.Port);
-                                            logger.LogDebug($"STUNClient Public IP={publicEndPoint.Address} Port={publicEndPoint.Port}.");

```

Comment:
If it were me, I would remove this line.  I think the logging gets done in either case on line 97.

Reply:
I was wondering why it was red!